src/awsexplorer/activation.ts

```diff
@@ -162,6 +164,17 @@ async function registerAwsExplorerCommands(
         )
     )
 
+    context.subscriptions.push(
```

Comment:
Will we have more api gateway commands? It may be worth spinning up an api gateway "activate" file similar to other modules

Reply:
moved